dandi/organize.py

```diff
@@ -400,7 +400,7 @@ def _sanitize_value(value, field):
     Of particular importance is _ which we use, as in BIDS, to separate
     _key-value entries
     """
-    value = re.sub("[_*:%@]", "-", value)
+    value = re.sub("[_*:%@;]", "-", value)
```

Comment:
```suggestion
    value = re.sub("[_*\/<>:|\"?*%@;]", "-", value)
```
expanding with suggested by @jwodder as not allowed on windows